frontend/js/behaviors/core/primaryNavigation.js

```diff
@@ -103,15 +99,6 @@ export default function(container) {
         }
     }
 
-    function isLevelOneMenuItem(menuItem) {
-        for (let index in levelOneMenuItems) {
```

Comment:
It looks like `levelOneMenuItems` isn't used anymore, so we should delete the variable declaration on line 29:
https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=666&id=art-institute-of-chicago_artic.edu&open=AZSKczYHH30yC7627MpE